integration-tests/utils/integration-utils.js

```diff
@@ -362,18 +362,22 @@ const withScreenshot = _.curry((testName, fn) => async options => {
 
 // Emitted when the page crashes
 const logError = page => {
-  const handle = msg => {
-    console.error('page.error', typeof msg === 'object' ? JSON.stringify(msg) : msg)
-  }
+  // this error will log an object of type [Error: Response] which is not very
```

Comment:
Slight clarification... this happens only when our Ajax code throws a Response object after an error response from a request. For other types of errors, the output will be different.